packages/react-components/react-menu/etc/react-menu.api.md

```diff
@@ -4,13 +4,13 @@
 
 ```ts
 
-import { ComponentProps } from '@fluentui/react-utilities';
-import { ComponentState } from '@fluentui/react-utilities';
+import type { ComponentProps } from '@fluentui/react-utilities';
```

Comment:
Good addition of the type. Nit: ComponentProps and ComponentState imports can be merged as they are both coming from react-utilities.